### PR TITLE
Interrogating neural networks lesson #418

### PR DESCRIPTION
**Pull request by hawc2**
Hi @anisa-hawes, the lesson, Interrogating a National Narrative with Recurrent Neural Networks, should be ready to go

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] ~~if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description~~
- [x] Add the appropriate "Label"
- [ ] [Ensure the status checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [ ] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [ ] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing build errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Build Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-build-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by hawc2** [comment]
the only issue I see @anisa-hawes is that the abstract is way longer than usual, right? should it be condensed?

**Comment by hawc2** [comment]
looks like there's an error, but it might be the same problem with the html-proofer?

**Comment by hawc2** [comment]
I fixed one mistaken link in the lesson, but the other htmlproofer errors seem to be related to an external link from another lesson. @ZoeLeBlanc just want to alert you to this as well.  This is the relevant error message

```
Found 685 internal links in the cache...
- ./_site/en/lessons/building-static-sites-with-jekyll-github-pages.html
Ran on 414 files!
  *  External link https://elotroalex.github.io/ed/documentation failed: 404 No error

- ./_site/es/lecciones/sitios-estaticos-con-jekyll-y-github-pages.html
  *  External link https://elotroalex.github.io/ed/ failed: 404 No error

htmlproofer 3.17.4 | Error:  HTML-Proofer found 2 failures!
```

**Comment by ZoeLeBlanc** [comment]
Thanks for the heads up @hawc2! Feel free to either fix those errors in this lesson with this PR, or we could open a separate PR. Thoughts @anisa-hawes ?

**Comment by anisa-hawes** [comment]
I'll go ahead and fix [this one](https://elotroalex.github.io/ed/documentation) in a separate issue. Done #2697.

**Comment by anisa-hawes** [comment]
> the only issue I see @anisa-hawes is that the abstract is way longer than usual, right? should it be condensed?

Yes, I agree the abstract could be shortened and simplified.

Even something as short as: _In this lesson, you will learn how to apply a Generative Pre-trained Transformer language model to a large-scale corpus so that you can locate broad themes and trends within written text._

**Comment by hawc2** [comment]
That sounds good to me, do you want to make that edit or should I?

**Comment by anisa-hawes** [comment]
Okay. Just reviewing this in Netlify, and the key problem I note is that the JISC banner doesn't display ! 😞 

We need to take another look at #2616 or hard-code it in this time.

**Review by anisa-hawes — CHANGES_REQUESTED** [review]
Reviewing this in Netlify, and the key problem I note is that the JISC banner doesn't display ! 😞

We need to take another look at https://github.com/programminghistorian/jekyll/pull/2616 or hard-code it in this time.

**Comment by ZoeLeBlanc** [comment]
Unless I'm mistaken there's no frontmatter indicating that this lesson is part of the partnership so until we add that the banner won't display @anisa-hawes (am busy atm but can work on this later tonight)

**Comment by hawc2** [comment]
Looks good to go! Thanks @ZoeLeBlanc and @anisa-hawes. Anisa, can you approve and merge?

**Review by anisa-hawes — DISMISSED** [review]

**Comment by anisa-hawes** [comment]
I'm happy with this! 

Just holding off from merging until I have confirmation that @drjwbaker is satisfied with the partnership page, or if he would like us to add more text explaining the commission/selection/editing process. 

I'm also organising some coordinated Twitter celebrations, as this will be the first lesson in our JISC/TNA series to be published.

**Comment by anisa-hawes** [comment]
Hello @hawc2 You’ll see that I have made a few small changes to the banner/partnership pages after chatting with James. 

I have one technical question remaining about the function of the partnership landing pages (/en/jisc-tna-partnership, es/jisc-tna-colaboracion, fr/jisc-tna-partenariat and pt/jisc-tna-parceria): I understood our ambition to be for each landing page (EN, ES, FR and PT) to provide a complete, combined list of all lessons in any language(s) that contain this partnership field within their YAML. 

But in this netlify preview, only the en/ page is available. I expected to be able to click through to the ES, FR and PT landing pages and see this /en lesson listed as the first to be published within this series. 

Perhaps I am misunderstanding how the landing page is set up. Are you able to advise, @ZoeLeBlanc? 

cc. @jenniferisasi

**Comment by hawc2** [comment]
@anisa-hawes I'm guessing you only see en/ because that's the only language in which this lesson has been published. 

Either way, can we please move forward with publishing this lesson? Are we waiting on @drjwbaker approval of the partnership page or is there something else that remains to be done? Everything looks good to go to me, and we can always improve the JISC banner later.

**Comment by drjwbaker** [comment]
No more approvals needed from me. And like @hawc2, I'm happy for this to go out before we've fully sorted out the partnership landing page logic (presuming it doesn't bork the whole site!)

**Comment by hawc2** [comment]
ok then @anisa-hawes just needs to approve the pull request and we should be good to publish!

**Review by anisa-hawes — APPROVED** [review]

**Comment by anisa-hawes** [comment]
Thank you for your patience, @hawc2. Zoe & I are working on some tweaks in a separate branch which will ensure consistent use of the YAML fields `redirect_from:` and `original:` across various pages of our website including the partnership landing pages (/en/jisc-tna-partnership, es/jisc-tna-colaboracion, fr/jisc-tna-partenariat and pt/jisc-tna-parceria).
